dronesym-frontend/src/app/app.module.ts

```diff
@@ -43,6 +44,7 @@ import { BreadcrumbComponent } from './breadcrumb/breadcrumb.component';
     DroneListComponent,
     DronesBoxComponent,
     UserManagementComponent,
+    SignupComponent
```

Comment:
Angular throws an error in here.
 , is missing after SignUpComponent
@meetcric

Reply:
oo, sorry it missed out while rebasing @dilinade i will fix it .